phpunit/class-rest-widgets-controller-test.php

```diff
@@ -219,31 +268,78 @@ public function test_get_items() {
 		$this->assertEqualSets(
 			array(
 				array(
-					'id'           => 'text-1',
+					'id'           => 'block-1',
 					'sidebar'      => 'sidebar-1',
 					'settings'     => array(
-						'text' => 'Custom text test',
+						'content' => '<!-- wp:paragraph --><p>Block test</p><!-- /wp:paragraph -->',
```

Comment:
I'd pull this repeated string into a variable.

Reply:
👍 ea730340e086bd7019acd4ef98a60e660cb18c58